v22.2/rename-constraint.md

```diff
@@ -23,22 +23,22 @@ It is not possible to rename a constraint for a column referenced by a view. For
 
 ## Required privileges
```

Comment:
If the contents of this section are all being commented out, you should comment out the header, too.